code/game/machinery/scanner_gate.dm

```diff
@@ -378,6 +390,14 @@
 						detect_gender = "female"
 			. = TRUE
 		//SKYRAT EDIT END - MORE SCANNER GATE OPTIONS
+		// BUBBER EDIT ADDITION BEGIN - NANITES
+		if("set_nanite_cloud")
+			var/new_cloud_id = params["new_cloud"]
+			if(!isnum(new_cloud_id))
+				return
+			nanite_cloud = clamp(new_cloud_id, 1, 100)
```

Comment:
Can you use a shared define for max and min clouds?

Reply:
would need to change it in the other code and pass it in static data so the chamber control panel can use it too, that alright with you?